src/components/Footer.tsx

```diff
@@ -76,22 +76,22 @@ function Footer() {
       <div className="footer-div footer-div-space">
         <h2>Support</h2>
         <p>
-          <a className="footer-text" href="/">
+          <a className="footer-text" href="/ComingSoon">
             Privacy Policy
           </a>
         </p>
         <p>
           <a
             className="footer-text"
-            href="https://www.linkedin.com/company/vito-research/about/"
+            href="mailto:andreasink17@gmail.com"
           >
             Contact
           </a>
         </p>
         <p>
           <a
             className="footer-text"
-            href="https://www.craft.do/s/jK9mX5dA3exvRw"
+            href="/ComingSoon"
           >
             Terms and Conditions
           </a>
```

Comment:
Use the `Link` component from react-router-dom for navigating to pages within the site